### ISSUE TITLE
Search ContentItem Actions/Menus

### ISSUE THREAD

**Issue by cmontalban** [daisy/DAISY-Online-Refresh#95]
The Best practice guide shows an example of buying books.
It asks what book would you like to buy and then presents a list.

I would suggest it would be more natural to retrieve a list of items in a ContentList and then be able to ask GetQuestions for a given ContentId

For example, if I want a user to be able to add a Book to a Wishlist/Favourites then this could be supported on a ContentId level GetQuestions.

Object/Action - Locate the Book using Search and then show me what actions I can perform on that Object.

This would require an extension to allow for the passing of a ContentId
This could be done with a very simple change to the spec adding a QuestionId token of 'ContentItem' with a value of the contentId.

Suggestion: Implement ContentItem token for getQuestions.

**Comment by cmontalban** [comment]
Resolved in issue #96

**Comment by jmkok** [comment]
I guess you want to add "contentId" as a [reserved id](http://www.daisy.org/projects/daisy-online-delivery/drafts/20100402/do-spec-20100402.html#fundamentals_dynamic_menus_reservedID)

That is fine, but actually not necessary. A search is not required to return a "contentListRef". A search may simply return a "multipleChoiceQuestion" response, which gives full control of the choices.

See example below (copy, modified and pasted from Aangepast Lezen)

Request
```xml
<?xml version="1.0" encoding="UTF-8"?>
<SOAP-ENV:Envelope xmlns:SOAP-ENV="http://schemas.xmlsoap.org/soap/envelope/" xmlns:ns1="http://www.daisy.org/ns/daisy-online/">
<SOAP-ENV:Body>
<ns1:getQuestions>
	<ns1:userResponses>
		<ns1:userResponse questionID="search_title" value="test"/>
	</ns1:userResponses>
</ns1:getQuestions>
</SOAP-ENV:Body>
</SOAP-ENV:Envelope>
```

Response
```xml
<s:Envelope xmlns:s="http://schemas.xmlsoap.org/soap/envelope/">
<s:Body xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
<getQuestionsResponse xmlns="http://www.daisy.org/ns/daisy-online/">
	<questions>
		<multipleChoiceQuestion id="content_information">
		<label xml:lang="nl">
			<text>U heeft Test gekozen. Kies het boek van uw keuze.</text>
		</label>
		<choices>
			<choice id="a441748">
				<label xml:lang="nl">
				<text>De vijand komt van opzij</text>
				<audio uri="http://server.com/000_De_vijand_komt_van_opzi.mp3" rangeBegin="0" rangeEnd="45685" size="45685"/>
				</label>
			</choice>
			<choice id="a441980">
				<label xml:lang="nl">
				<text>Als je het licht niet kunt zien</text>
				<audio uri="http://server.com/something/000_Als_je_het_licht_niet_k.mp3" rangeBegin="0" rangeEnd="42341" size="42341"/>
				</label>
			</choice>
			<choice id="a442520">
				<label xml:lang="nl">
				<text>Bloemen langs de weg</text>
				<audio uri="http://server.com/something/000_Bloemen_langs_de_weg_De.mp3" rangeBegin="0" rangeEnd="109424" size="109424"/>
				</label>
			</choice>
		</choices>
		</multipleChoiceQuestion>
	</questions>
</getQuestionsResponse>
</s:Body>
</s:Envelope>
```
